.travis.yml

```diff
@@ -162,3 +162,6 @@ jobs:
                     tags: true
             after_deploy:
                 - ./dev-tools/trigger-website.sh ${TRAVIS_TOKEN} ${TRAVIS_TAG}
+
+    allow_failures:
+        -   php: 7.4snapshot
```

Comment:
This should be reverted, see #4559.

Reply:
No problem to remove the commit, but it will make the build fail / unstable I fear.